database/__init__.py

```diff
@@ -0,0 +1,45 @@
+from pyramid.config import Configurator
+
+
+from sqlalchemy import engine_from_config
+
+
+from .models import DBSession, Base, Node
+
+
+def main(global_config, **settings):
```

Comment:
global_config is not used here. What is it meant to be used for? Can we make use of it somewhere?

Reply:
This comes from pyramid. It passes around a configurator instance when loading the app.